### PR TITLE
Hyperlink text color for mobile changed.

### PR DESCRIPTION
**Pull request by mashrin**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -70,10 +70,10 @@
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right" style="font-family: 'Montserrat', sans-serif;">
                     <li>
-                        <a class="page-scroll" href="aboutus/index.php">About</a>
+                        <a class="page-scroll" href="aboutus/index.php" style="color: rgb(224,255,255)">About</a>
```
Comment (prastut) [inline index.php]: - Don't write inline styles. Merge them with css. 
- Don't use `rgb`. Use `hex` based colors. 
- This makes changes at all screen sizes (Even at desktop too). I need these changes only at mobile.

File: index.php [hunk 2]
```diff
@@ -554,4 +554,4 @@ function hello(msg)
 if (isset ($conn)){
     mysqli_close($conn);
 }
-?>
+?>
```
Comment (prastut) [inline index.php]: Have a whitespace at the end of the file. Don't remove.

File: aboutus/index.php [hunk 3]
```diff
@@ -771,10 +771,10 @@
                     <div id="form-div">
                         <form action="index.php" method="post" class="form" id="form1">
                             <p class="name">
-                                <input name="name" type="text" class="validate[required,custom[onlyLetter],length[0,100]] feedback-input" placeholder="Name" id="name" />
+                                <input name="name" type="text" class="validate[required,custom[onlyLetter],length[0,100]] feedback-input" placeholder="       Name" id="name" />
```
Comment (prastut) [inline aboutus/index.php]: - This is a hackish way to go about it. 
- We care about code as much as we care about design. 
- Use CSS to your aid.

**Review by prastut — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (mashrin) [reply index.php]: @prastut will correct it soon.

And on desktop the color anyway was the same. Isn't it?

**Pull request closed without merging**